company/welcome/other-roles.md

```diff
@@ -0,0 +1,9 @@
+# Non-Engineers
```

Comment:
Looking at the content of this it looks like it's sales specific rather than all non-engineer roles